key-share/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.2.2
+* Add `no_std` support [#92]
+
+[#92]: https://github.com/dfns/cggmp21/pull/92
+
```

Comment:
We forgot to update the changelog on that MR? Oh no

Reply:
We need a workflow!